LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2015–17 Hack Cambridge
```

Comment:
Copyright year should be a comma separated list of the year of first publication and each year in which modifications were made. First publication for this repo was 2016